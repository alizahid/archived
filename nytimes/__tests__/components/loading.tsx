import React from 'react'
import { create } from 'react-test-renderer'

import Loading from '../../src/components/loading'

it('renders', () => {
  const tree = create(<Loading />)

  expect(tree).toMatchSnapshot()
})

it('null on Android', () => {
  jest.doMock('Platform', () => ({
    OS: 'android'
  }))

  const tree = create(<Loading />).toJSON()

  expect(tree).toBe(null)
})
