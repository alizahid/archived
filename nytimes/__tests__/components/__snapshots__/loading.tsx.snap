// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
      "margin": 20,
    }
  }
>
  <ActivityIndicator
    animating={true}
    color="#4CD964"
    hidesWhenStopped={true}
    size="large"
  />
</View>
`;
